yardoc_frontpage.md

```diff
@@ -1,24 +1,24 @@
-# Welcome to the SolarWindsAPM Ruby Gem
 
+# Welcome to the SolarWindsAPM Ruby Gem
```

Comment:
Curious, how/where is this yardoc page used?

Reply:
Although, the file yardoc_frontpage.md is used in .yardopts (point yardoc_frontpage.md as default readme), but since [appoptics](https://www.rubydoc.info/gems/appoptics_apm), rubydoc.info doesn't use the yardoc_frontpage.md as default readme.

Reply:
If you're finding that it's not used anymore feel free to delete :)

Reply:
Do we need some sort of welcome page for rubydoc.info? or the current README is sufficient?

Reply:
This is our current https://rubydoc.info/github/solarwinds/apm-ruby page you're referring to, yes? IMHO it is enough so we don't need the yardoc_frontpage content.

Reply:
Updated.